lib/Dialect/TorchConversion/Transforms/Passes.cpp

```diff
@@ -18,7 +18,9 @@
 #include "torch-mlir/Conversion/TorchToSCF/TorchToSCF.h"
 #include "torch-mlir/Conversion/TorchToTMTensor/TorchToTMTensor.h"
 #include "torch-mlir/Conversion/TorchToTensor/TorchToTensor.h"
+#ifdef TORCH_MLIR_ENABLE_TOSA
```

Comment:
How about moving both `#ifdef`s to a separate include block? Right now they are in the middle of other includes, I guess to keep them alphabetical?

```c++
#include "torch-mlir/Conversion/TorchToTMTensor/TorchToTMTensor.h"
#include "torch-mlir/Conversion/TorchToTensor/TorchToTensor.h"
#include "torch-mlir/Dialect/Torch/Transforms/Passes.h"

#ifdef TORCH_MLIR_ENABLE_TOSA
#include "torch-mlir/Conversion/TorchToTosa/TorchToTosa.h"
#endif

#ifdef TORCH_MLIR_ENABLE_STABLEHLO
#include "stablehlo/transforms/Passes.h"
#include "torch-mlir/Conversion/TorchToStablehlo/TorchToStablehlo.h"
#endif
```

Reply:
I was being paranoid. A bunch of `#include` lines in C++ is like a house of cards.  But since you ask for it...

Reply:
If the ordering is load bearing, that's worth documenting too. I share your paranoia :D